rasa_core/policies/ensemble.py

```diff
@@ -249,12 +249,11 @@ def from_dict(cls, dictionary: Dict[Text, Any]) -> List[Policy]:
                 # override policy's featurizer with real featurizer class
                 policy['featurizer'] = featurizer_func(**featurizer_config)
 
-            constr_func = utils.class_from_module_path(policy_name)
-
-            if constr_func:
+            try:
+                constr_func = utils.class_from_module_path(policy_name)
                 policy_object = constr_func(**policy)
                 parsed_policies.append(policy_object)
-            else:
+            except(ImportError):
```

Comment:
this also needs to accept an AttributeError, did you try the case where you specify the policy as "test.TestPolicy"? maybe write a test for this as well if one doesn't exist yet

Reply:
Not quite sure what you mean as I don't know the workings of loading policies yet.
How would this be different from the `test_invalid_policy_configurations` that we already have?